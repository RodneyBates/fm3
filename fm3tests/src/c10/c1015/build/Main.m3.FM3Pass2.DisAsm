 ItkEOF()
 ItkLeftEnd()
 ItkModuleLt(L2(16_2),[1,0])
 ItkDeclScopeLt(L1(16_1))
 ItkOpenScopeLt(L1(16_1))
 ItkOpenDeclListLt(L19(16_13),[5,2])
 ItkConstDeclLt([5,8])
 ItkDeclNo(D3,[5,8])
 ItkConstDeclType([5,10])
 ItkDeclTypeAbsent([5,10])
 ItkConstDeclValue([5,10])
 ItkDefTopExprNo(L52(16_34),[5,12])
 ItkConstDeclRt([5,8])
 ItkOpenDeclListSep(L1(16_1),[5,2])
 ItkConstDeclLt([6,8])
 ItkDeclNo(D4,[6,8])
 ItkConstDeclType([6,10])
 ItkDeclTypeAbsent([6,10])
 ItkConstDeclValue([6,10])
 ItkDefTopExprNo(L49(16_31),[6,12])
 ItkConstDeclRt([6,8])
 ItkOpenDeclListSep(L2(16_2),[5,2])
 ItkConstDeclLt([7,8])
 ItkDeclNo(D5,[7,8])
 ItkConstDeclType([7,10])
 ItkDeclTypeAbsent([7,10])
 ItkConstDeclValue([7,10])
 ItkDefTopExprNo(L47(16_2f),[7,12])
 ItkConstDeclRt([7,8])
 ItkOpenDeclListSep(L3(16_3),[5,2])
 ItkConstDeclLt([9,8])
 ItkDeclNo(D6,[9,8])
 ItkConstDeclType([9,10])
 ItkDeclTypeAbsent([9,10])
 ItkConstDeclValue([9,10])
 ItkDefTopExprNo(L44(16_2c),[9,12])
 ItkConstDeclRt([9,8])
 ItkOpenDeclListSep(L4(16_4),[5,2])
 ItkConstDeclLt([10,8])
 ItkDeclNo(D7,[10,8])
 ItkConstDeclType([10,10])
 ItkDeclTypeAbsent([10,10])
 ItkConstDeclValue([10,10])
 ItkDefTopExprNo(L41(16_29),[10,12])
 ItkConstDeclRt([10,8])
 ItkOpenDeclListSep(L5(16_5),[5,2])
 ItkConstDeclLt([11,8])
 ItkDeclNo(D8,[11,8])
 ItkConstDeclType([11,10])
 ItkDeclTypeAbsent([11,10])
 ItkConstDeclValue([11,10])
 ItkDefTopExprNo(L38(16_26),[11,12])
 ItkConstDeclRt([11,8])
 ItkOpenDeclListSep(L6(16_6),[5,2])
 ItkConstDeclLt([12,8])
 ItkDeclNo(D9,[12,8])
 ItkConstDeclType([12,10])
 ItkDeclTypeAbsent([12,10])
 ItkConstDeclValue([12,10])
 ItkDefTopExprNo(L35(16_23),[12,12])
 ItkConstDeclRt([12,8])
 ItkOpenDeclListSep(L7(16_7),[5,2])
 ItkConstDeclLt([13,8])
 ItkDeclNo(D10,[13,8])
 ItkConstDeclType([13,10])
 ItkDeclTypeAbsent([13,10])
 ItkConstDeclValue([13,10])
 ItkDefTopExprNo(L32(16_20),[13,12])
 ItkConstDeclRt([13,8])
 ItkOpenDeclListSep(L8(16_8),[5,2])
 ItkConstDeclLt([14,8])
 ItkDeclNo(D11,[14,8])
 ItkConstDeclType([14,10])
 ItkDeclTypeAbsent([14,10])
 ItkConstDeclValue([14,10])
 ItkDefTopExprNo(L29(16_1d),[14,12])
 ItkConstDeclRt([14,8])
 ItkOpenDeclListSep(L9(16_9),[5,2])
 ItkConstDeclLt([15,8])
 ItkDeclNo(D12,[15,8])
 ItkConstDeclType([15,10])
 ItkDeclTypeAbsent([15,10])
 ItkConstDeclValue([15,10])
 ItkDefTopExprNo(L26(16_1a),[15,12])
 ItkConstDeclRt([15,8])
 ItkOpenDeclListSep(L10(16_a),[5,2])
 ItkConstDeclLt([17,8])
 ItkDeclNo(D13,[17,8])
 ItkConstDeclType([17,10])
 ItkDeclTypeAbsent([17,10])
 ItkConstDeclValue([17,10])
 ItkDefTopExprNo(L23(16_17),[17,12])
 ItkConstDeclRt([17,8])
 ItkOpenDeclListSep(L11(16_b),[5,2])
 ItkConstDeclLt([18,8])
 ItkDeclNo(D14,[18,8])
 ItkConstDeclType([18,10])
 ItkDeclTypeAbsent([18,10])
 ItkConstDeclValue([18,10])
 ItkDefTopExprNo(L20(16_14),[18,12])
 ItkConstDeclRt([18,8])
 ItkOpenDeclListSep(L12(16_c),[5,2])
 ItkConstDeclLt([19,8])
 ItkDeclNo(D15,[19,8])
 ItkConstDeclType([19,10])
 ItkDeclTypeAbsent([19,10])
 ItkConstDeclValue([19,10])
 ItkDefTopExprNo(L17(16_11),[19,12])
 ItkConstDeclRt([19,8])
 ItkOpenDeclListSep(L13(16_d),[5,2])
 ItkConstDeclLt([21,8])
 ItkDeclNo(D16,[21,8])
 ItkConstDeclType([21,10])
 ItkDeclTypeAbsent([21,10])
 ItkConstDeclValue([21,10])
 ItkDefTopExprNo(L14(16_e),[21,12])
 ItkConstDeclRt([21,8])
 ItkOpenDeclListSep(L14(16_e),[5,2])
 ItkConstDeclLt([22,8])
 ItkDeclNo(D17,[22,8])
 ItkConstDeclType([22,10])
 ItkDeclTypeAbsent([22,10])
 ItkConstDeclValue([22,10])
 ItkDefTopExprNo(L11(16_b),[22,12])
 ItkConstDeclRt([22,8])
 ItkOpenDeclListSep(L15(16_f),[5,2])
 ItkConstDeclLt([23,8])
 ItkDeclNo(D18,[23,8])
 ItkConstDeclType([23,10])
 ItkDeclTypeAbsent([23,10])
 ItkConstDeclValue([23,10])
 ItkDefTopExprNo(L8(16_8),[23,12])
 ItkConstDeclRt([23,8])
 ItkOpenDeclListSep(L16(16_10),[5,2])
 ItkConstDeclLt([24,8])
 ItkDeclNo(D19,[24,8])
 ItkConstDeclType([24,10])
 ItkDeclTypeAbsent([24,10])
 ItkConstDeclValue([24,10])
 ItkDefTopExprNo(L5(16_5),[24,12])
 ItkConstDeclRt([24,8])
 ItkOpenDeclListSep(L17(16_11),[5,2])
 ItkConstDeclLt([26,8])
 ItkDeclNo(D20,[26,8])
 ItkConstDeclType([26,10])
 ItkDeclTypeAbsent([26,10])
 ItkConstDeclValue([26,10])
 ItkDefTopExprNo(L3(16_3),[26,12])
 ItkConstDeclRt([26,8])
 ItkOpenDeclListSep(L18(16_12),[5,2])
 ItkConstDeclLt([27,8])
 ItkDeclNo(D21,[27,8])
 ItkConstDeclType([27,10])
 ItkDeclTypeAbsent([27,10])
 ItkConstDeclValue([27,10])
 ItkDefTopExprNo(L1(16_1),[27,12])
 ItkConstDeclRt([27,8])
 ItkOpenDeclListSep(L19(16_13),[5,2])
 ItkOpenDeclListRt(L19(16_13),[5,2])
 ItkModuleBegin(L2(16_2),[31,2])
 ItkStmtListLt(L0(16_0),[32,2])
 ItkStmtListRt(L0(16_0),[32,2])
 ItkOpenScopeRt(L1(16_1))
 ItkDeclScopeRt(L1(16_1))
 ItkModuleRt(L2(16_2),[1,0])
 ItkRightEnd()
 ItkBOF()
     0
