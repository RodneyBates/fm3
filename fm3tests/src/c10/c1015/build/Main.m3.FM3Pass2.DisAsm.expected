 ItkEOF()
 ItkLeftEnd()
 ItkModuleLt(L1(16_1),[1,0])
 ItkBlockDeclListLt(L20(16_14),[5,2])
 ItkConstDeclLt([5,8])
 ItkDeclNo(D1,[5,8])
 ItkConstDeclColon([5,10],FALSE)
 ItkDeclTypeAbsent()
 ItkConstDeclEquals([5,10])
 ItkORLt([5,14])
 ItkIdRefDeclNo(D2,[5,12])
 ItkORBinOp([5,14])
 ItkIdRefDeclNo(D3,[5,17])
 ItkORRt([5,14])
 ItkConstDeclRt([5,8])
 ItkBlockDeclListSep(L1(16_1),[6,0])
 ItkConstDeclLt([6,8])
 ItkDeclNo(D2,[6,8])
 ItkConstDeclColon([6,10],FALSE)
 ItkDeclTypeAbsent()
 ItkConstDeclEquals([6,10])
 ItkANDLt([6,14])
 ItkIdRefDeclNo(D4,[6,12])
 ItkANDBinOp([6,14])
 ItkIdRefDeclNo(D5,[6,18])
 ItkANDRt([6,14])
 ItkConstDeclRt([6,8])
 ItkBlockDeclListSep(L2(16_2),[7,0])
 ItkConstDeclLt([7,8])
 ItkDeclNo(D3,[7,8])
 ItkConstDeclColon([7,10],FALSE)
 ItkDeclTypeAbsent()
 ItkConstDeclEquals([7,10])
 ItkNOTLt([7,12])
 ItkIdRefDeclNo(D6,[7,16])
 ItkNOTRt([7,12])
 ItkConstDeclRt([7,8])
 ItkBlockDeclListSep(L3(16_3),[9,0])
 ItkConstDeclLt([9,8])
 ItkDeclNo(D4,[9,8])
 ItkConstDeclColon([9,10],FALSE)
 ItkDeclTypeAbsent()
 ItkConstDeclEquals([9,10])
 ItkEqualLt([9,14])
 ItkIdRefDeclNo(D7,[9,12])
 ItkEqualBinOp([9,14])
 ItkIdRefDeclNo(D8,[9,16])
 ItkEqualRt([9,14])
 ItkConstDeclRt([9,8])
 ItkBlockDeclListSep(L4(16_4),[10,0])
 ItkConstDeclLt([10,8])
 ItkDeclNo(D5,[10,8])
 ItkConstDeclColon([10,10],FALSE)
 ItkDeclTypeAbsent()
 ItkConstDeclEquals([10,10])
 ItkUnequalLt([10,14])
 ItkIdRefDeclNo(D9,[10,12])
 ItkUnequalBinOp([10,14])
 ItkIdRefDeclNo(D10,[10,16])
 ItkUnequalRt([10,14])
 ItkConstDeclRt([10,8])
 ItkBlockDeclListSep(L5(16_5),[11,0])
 ItkConstDeclLt([11,8])
 ItkDeclNo(D6,[11,8])
 ItkConstDeclColon([11,10],FALSE)
 ItkDeclTypeAbsent()
 ItkConstDeclEquals([11,10])
 ItkLessLt([11,14])
 ItkIdRefDeclNo(D11,[11,12])
 ItkLessBinOp([11,14])
 ItkIdRefDeclNo(D12,[11,16])
 ItkLessRt([11,14])
 ItkConstDeclRt([11,8])
 ItkBlockDeclListSep(L6(16_6),[12,0])
 ItkConstDeclLt([12,8])
 ItkDeclNo(D7,[12,8])
 ItkConstDeclColon([12,10],FALSE)
 ItkDeclTypeAbsent()
 ItkConstDeclEquals([12,10])
 ItkLessEqualLt([12,14])
 ItkIdRefDeclNo(D13,[12,12])
 ItkLessEqualBinOp([12,14])
 ItkIdRefDeclNo(D14,[12,17])
 ItkLessEqualRt([12,14])
 ItkConstDeclRt([12,8])
 ItkBlockDeclListSep(L7(16_7),[13,0])
 ItkConstDeclLt([13,8])
 ItkDeclNo(D8,[13,8])
 ItkConstDeclColon([13,10],FALSE)
 ItkDeclTypeAbsent()
 ItkConstDeclEquals([13,10])
 ItkGreaterLt([13,14])
 ItkIdRefDeclNo(D15,[13,12])
 ItkGreaterBinOp([13,14])
 ItkIdRefDeclNo(D16,[13,16])
 ItkGreaterRt([13,14])
 ItkConstDeclRt([13,8])
 ItkBlockDeclListSep(L8(16_8),[14,0])
 ItkConstDeclLt([14,8])
 ItkDeclNo(D9,[14,8])
 ItkConstDeclColon([14,10],FALSE)
 ItkDeclTypeAbsent()
 ItkConstDeclEquals([14,10])
 ItkGreaterEqualLt([14,14])
 ItkIdRefDeclNo(D17,[14,12])
 ItkGreaterEqualBinOp([14,14])
 ItkIdRefDeclNo(D18,[14,17])
 ItkGreaterEqualRt([14,14])
 ItkConstDeclRt([14,8])
 ItkBlockDeclListSep(L9(16_9),[15,0])
 ItkConstDeclLt([15,8])
 ItkDeclNo(D10,[15,8])
 ItkConstDeclColon([15,10],FALSE)
 ItkDeclTypeAbsent()
 ItkConstDeclEquals([15,10])
 ItkINLt([15,14])
 ItkIdRefDeclNo(D19,[15,12])
 ItkINBinOp([15,14])
 ItkIdRefDeclNo(D20,[15,17])
 ItkINRt([15,14])
 ItkConstDeclRt([15,8])
 ItkBlockDeclListSep(L10(16_a),[17,0])
 ItkConstDeclLt([17,8])
 ItkDeclNo(D11,[17,8])
 ItkConstDeclColon([17,10],FALSE)
 ItkDeclTypeAbsent()
 ItkConstDeclEquals([17,10])
 ItkBinaryPlusLt([17,14])
 ItkIdRefDeclNo(D1,[17,12])
 ItkBinaryPlusBinOp([17,14])
 ItkIdRefDeclNo(D2,[17,16])
 ItkBinaryPlusRt([17,14])
 ItkConstDeclRt([17,8])
 ItkBlockDeclListSep(L11(16_b),[18,0])
 ItkConstDeclLt([18,8])
 ItkDeclNo(D12,[18,8])
 ItkConstDeclColon([18,10],FALSE)
 ItkDeclTypeAbsent()
 ItkConstDeclEquals([18,10])
 ItkBinaryMinusLt([18,14])
 ItkIdRefDeclNo(D3,[18,12])
 ItkBinaryMinusBinOp([18,14])
 ItkIdRefDeclNo(D4,[18,16])
 ItkBinaryMinusRt([18,14])
 ItkConstDeclRt([18,8])
 ItkBlockDeclListSep(L12(16_c),[19,0])
 ItkConstDeclLt([19,8])
 ItkDeclNo(D13,[19,8])
 ItkConstDeclColon([19,10],FALSE)
 ItkDeclTypeAbsent()
 ItkConstDeclEquals([19,10])
 ItkAmpersandLt([19,14])
 ItkIdRefDeclNo(D5,[19,12])
 ItkAmpersandBinOp([19,14])
 ItkIdRefDeclNo(D6,[19,16])
 ItkAmpersandRt([19,14])
 ItkConstDeclRt([19,8])
 ItkBlockDeclListSep(L13(16_d),[21,0])
 ItkConstDeclLt([21,8])
 ItkDeclNo(D14,[21,8])
 ItkConstDeclColon([21,10],FALSE)
 ItkDeclTypeAbsent()
 ItkConstDeclEquals([21,10])
 ItkStarLt([21,14])
 ItkIdRefDeclNo(D7,[21,12])
 ItkStarBinOp([21,14])
 ItkIdRefDeclNo(D8,[21,16])
 ItkStarRt([21,14])
 ItkConstDeclRt([21,8])
 ItkBlockDeclListSep(L14(16_e),[22,0])
 ItkConstDeclLt([22,8])
 ItkDeclNo(D15,[22,8])
 ItkConstDeclColon([22,10],FALSE)
 ItkDeclTypeAbsent()
 ItkConstDeclEquals([22,10])
 ItkSlashLt([22,14])
 ItkIdRefDeclNo(D9,[22,12])
 ItkSlashBinOp([22,14])
 ItkIdRefDeclNo(D10,[22,16])
 ItkSlashRt([22,14])
 ItkConstDeclRt([22,8])
 ItkBlockDeclListSep(L15(16_f),[23,0])
 ItkConstDeclLt([23,8])
 ItkDeclNo(D16,[23,8])
 ItkConstDeclColon([23,10],FALSE)
 ItkDeclTypeAbsent()
 ItkConstDeclEquals([23,10])
 ItkDIVLt([23,14])
 ItkIdRefDeclNo(D11,[23,12])
 ItkDIVBinOp([23,14])
 ItkIdRefDeclNo(D12,[23,18])
 ItkDIVRt([23,14])
 ItkConstDeclRt([23,8])
 ItkBlockDeclListSep(L16(16_10),[24,0])
 ItkConstDeclLt([24,8])
 ItkDeclNo(D17,[24,8])
 ItkConstDeclColon([24,10],FALSE)
 ItkDeclTypeAbsent()
 ItkConstDeclEquals([24,10])
 ItkMODLt([24,14])
 ItkIdRefDeclNo(D13,[24,12])
 ItkMODBinOp([24,14])
 ItkIdRefDeclNo(D14,[24,18])
 ItkMODRt([24,14])
 ItkConstDeclRt([24,8])
 ItkBlockDeclListSep(L17(16_11),[26,0])
 ItkConstDeclLt([26,8])
 ItkDeclNo(D18,[26,8])
 ItkConstDeclColon([26,10],FALSE)
 ItkDeclTypeAbsent()
 ItkConstDeclEquals([26,10])
 ItkUnaryPlusLt([26,12])
 ItkIdRefDeclNo(D15,[26,14])
 ItkUnaryPlusRt([26,12])
 ItkConstDeclRt([26,8])
 ItkBlockDeclListSep(L18(16_12),[27,0])
 ItkConstDeclLt([27,8])
 ItkDeclNo(D19,[27,8])
 ItkConstDeclColon([27,10],FALSE)
 ItkDeclTypeAbsent()
 ItkConstDeclEquals([27,10])
 ItkUnaryMinusLt([27,12])
 ItkIdRefDeclNo(D16,[27,14])
 ItkUnaryMinusRt([27,12])
 ItkConstDeclRt([27,8])
 ItkBlockDeclListSep(L19(16_13),[29,0])
 ItkConstDeclLt([29,8])
 ItkDeclNo(D20,[29,8])
 ItkConstDeclColon([29,10],FALSE)
 ItkDeclTypeAbsent()
 ItkConstDeclEquals([29,10])
 ItkIdRefDeclNo(D17,[29,12])
 ItkConstDeclRt([29,8])
 ItkBlockDeclListRt(L20(16_14),[5,2])
 ItkModuleBegin(L1(16_1),[31,2])
 ItkStmtListLt(L0(16_0),[32,2])
 ItkStmtListRt(L0(16_0),[32,2])
 ItkModuleRt(L1(16_1),[1,0])
 ItkRightEnd()
 ItkBOF()
     0
